http-clients/src/main/java/org/triplea/http/client/lobby/HttpLobbyClient.java

```diff
@@ -2,15 +2,21 @@
 
 import java.net.URI;
 import lombok.Getter;
+import org.triplea.http.client.lobby.game.listing.GameListingClient;
 import org.triplea.http.client.lobby.moderator.toolbox.HttpModeratorToolboxClient;
 
 /** Holder class for the various http clients that access lobby resources. */
 @Getter
 public class HttpLobbyClient {
   private final HttpModeratorToolboxClient httpModeratorToolboxClient;
-  // TODO: Project#12 Add additional http clients
+  private final GameListingClient gameListingClient;
 
-  public HttpLobbyClient(final URI lobbyUri, final String apiKey) {
+  private HttpLobbyClient(final URI lobbyUri, final String apiKey) {
     httpModeratorToolboxClient = new HttpModeratorToolboxClient(lobbyUri, apiKey);
+    gameListingClient = GameListingClient.newClient(lobbyUri, apiKey);
+  }
+
+  public static HttpLobbyClient newClient(final URI lobbyUri, final String apiKey) {
+    return new HttpLobbyClient(lobbyUri, apiKey);
   }
```

Comment:
What's the reasoning for having the `newClient` method vs using the constructor?

Reply:
It's more consistent with the other the http clients, they nearly all have a factory method rather than public constructor

Reply:
Ok. I don't really see much value but not a big deal.